qcodes/instrument_drivers/Keysight/keysight_b220x.py

```diff
@@ -4,6 +4,7 @@
 
 from qcodes import VisaInstrument
 from qcodes.utils.validators import MultiType, Ints, Enum, Lists
+from typing import List, Tuple
 
 
 def post_execution_status_poll(func):
```

Comment:
as the mypy output of the CI suggests, you need to add type hints to this decorator. you can read about this online, but perhaps smth simple like `def post_execution_status_poll(func: Callable) -> Callable:` could work.